tests/99-compare_databases.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+
+load libs/shared_setup
+
+setup () {
+    # These env variables must exist to get the compare_databses tests executed.
+    required="LOCAL_CI_TESTS_DBLIBRARY LOCAL_CI_TESTS_DBTYPE LOCAL_CI_TESTS_DBHOST LOCAL_CI_TESTS_DBUSER LOCAL_CI_TESTS_DBPASS"
+    for var in ${required}; do
+        if [ -z "${!var}" ]; then
+            skip "some required variables are not defined"
+        fi
+    done
+    # Only supported database is mysqli.
+    if [[ "$LOCAL_CI_TESTS_DBTYPE" != "mysqli" ]]; then
+        skip "only mysqli dbtype is supported"
+    fi
+    # All right, populate the needed script variables.
+    export dblibrary=$LOCAL_CI_TESTS_DBLIBRARY
+    export dbtype=$LOCAL_CI_TESTS_DBTYPE
+    export dbhost1=$LOCAL_CI_TESTS_DBHOST
+    export dbuser1=$LOCAL_CI_TESTS_DBUSER
+    export dbpass1=$LOCAL_CI_TESTS_DBPASS
+
+    create_git_branch master 35d5053ba20432059b497d85e39175d356f44fb4
```

Comment:
I would've used a tagged version here rather than that hash (though its not so important). I only used a hash in 99-grunt_process.bats because we didn't have a tag available yet which would work